docs/sources/clients/promtail/configuration.md

```diff
@@ -983,6 +986,47 @@ The list of labels below are discovered when consuming kafka:
 
 To keep discovered labels to your logs use the [relabel_configs](#relabel_configs) section.
 
+### GELF
+
+The `gelf` block configures a GELF UDP listener allowing users to push
+logs to Promtail with the [GELF](https://docs.graylog.org/docs/gelf) protocol.
```

Comment:
Does this mean Promtail has a continuous connection with a server? What's happening when it fails?

Reply:
Promtail is the server.

Reply:
And there's no connection in UDP.

Reply:
Ah, it's been a long time that I've looked at UDP :see_no_evil: